powersimdata/network/constants/region/usa.py

```diff
@@ -206,7 +205,9 @@ def get_loadzone_mapping(zone):
     mapping["timezone2id"] = {
         t: set(i) for t, i in zone.groupby("time_zone").groups.items()
     }
-    mapping["loadzone2id"] = {l: i for l, i in zone.groupby("zone_name").groups.items()}
+    mapping["loadzone2id"] = {
+        l: i[0] for l, i in zone.groupby("zone_name").groups.items()
+    }
```

Comment:
This is the fix you mentioned where we previously had a single-element index but now we have just an int?

Reply:
Yes. Since there is a one-to-one relation between `zone_name` and `zone_id`, we don't want a `list`. This is what we are doing in `develop`.